### PR TITLE
feat: implement code for Cloud Bigtable tutorial

### LINKED ISSUE #85: Implement examples for Cloud Bigtable tutorial
https://cloud.google.com/functions/docs/tutorials/use-cloud-bigtable

### PR DESCRIPTION
**Pull request by coryan**
Fixes #85

### CODE REVIEW

File: examples/site/tutorial_cloud_bigtable/tutorial_cloud_bigtable.cc [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+// [START bigtable_functions_quickstart]
+#include <google/cloud/bigtable/table.h>
+#include <google/cloud/functions/http_request.h>
+#include <google/cloud/functions/http_response.h>
+#include <algorithm>
+#include <cctype>
+#include <iostream>
+#include <mutex>
+#include <optional>
```
Comment (devbww) [inline examples/site/tutorial_cloud_bigtable/tutorial_cloud_bigtable.cc:23]: Unused?

File: examples/site/tutorial_cloud_bigtable/tutorial_cloud_bigtable.cc [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START bigtable_functions_quickstart]
+#include <google/cloud/bigtable/table.h>
+#include <google/cloud/functions/http_request.h>
+#include <google/cloud/functions/http_response.h>
+#include <algorithm>
+#include <cctype>
+#include <iostream>
```
Comment (devbww) [inline examples/site/tutorial_cloud_bigtable/tutorial_cloud_bigtable.cc:21]: Unused?

File: examples/site/tutorial_cloud_bigtable/tutorial_cloud_bigtable.cc [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+#include <mutex>
+#include <optional>
+#include <sstream>
+#include <string>
+#include <string_view>
```
Comment (devbww) [inline examples/site/tutorial_cloud_bigtable/tutorial_cloud_bigtable.cc:26]: Unused?

**Review by devbww — APPROVED** [review]

File: examples/site/tutorial_cloud_bigtable/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# ~~~
+# Copyright 2020 Google LLC
```
Comment (devbww) [inline examples/site/tutorial_cloud_bigtable/CMakeLists.txt:2]: 2021?

**Review by devbww — APPROVED** [review]

File: examples/site/tutorial_cloud_bigtable/tutorial_cloud_bigtable.cc (hunk 1 above)
Comment (coryan) [reply examples/site/tutorial_cloud_bigtable/tutorial_cloud_bigtable.cc]: Thanks, removed.

File: examples/site/tutorial_cloud_bigtable/tutorial_cloud_bigtable.cc (hunk 2 above)
Comment (coryan) [reply examples/site/tutorial_cloud_bigtable/tutorial_cloud_bigtable.cc]: Removed.

File: examples/site/tutorial_cloud_bigtable/tutorial_cloud_bigtable.cc (hunk 3 above)
Comment (coryan) [reply examples/site/tutorial_cloud_bigtable/tutorial_cloud_bigtable.cc]: Removed.

File: examples/site/tutorial_cloud_bigtable/CMakeLists.txt (hunk 4 above)
Comment (coryan) [reply examples/site/tutorial_cloud_bigtable/CMakeLists.txt]: Fixed

**Pull request merged**